system/jlib/jlog.cpp

```diff
@@ -197,7 +197,7 @@ const char * LogMsgJobInfo::queryJobIDStr() const
     if (isDeserialized)
         return jobIDStr;
     else if (jobID == UnknownJob)
-        return "Unknown";
+        return "UNK";
```

Comment:
What's the rationale for this change? Job IDs are not normally 3 chars so it won't do anything for alignment...

Reply:
To stay consistent with other columns' output (albeit Audience and Class have known static widths) and honestly to cut back on unnecessary output (even though 4 chars per log entry isn't significant). It seems raw alignment will suffer regardless, unless we pad to known job ID widths, hopefully abstracted views of the logs depend on the column delimiter instead.